lib/native/io.ts

```diff
@@ -145,19 +145,23 @@ export class TlsContextOptions {
     /** Minimum version of TLS to support. Uses OS/system default if unspecified. */
     public min_tls_version: TlsVersion = TlsVersion.Default;
     /** Path to a single file with all trust anchors in it, in PEM format */
-    public ca_file?: string = undefined;
+    public ca_file?: string;
     /** Path to directory containing trust anchors. Only used on Unix-style systems. */
-    public ca_path?: string = undefined;
+    public ca_path?: string;
     /** Semi-colon delimited list of ALPN protocols to be used on platforms which support ALPN */
-    public alpn_list?: string = undefined;
+    public alpn_list?: string;
```

Comment:
don't do it in this PR
but maybe you want to copy what I did today in python:
- rename `ca_path` to `ca_dirpath`
- rename all the other `_path` or `_file` args/vars to `_filepath`
- make `alpn_list` an actual list instead of semicolon delimited string

Reply:
That sounds like a magnificent follow up